addons/mail/models/mail_activity_type.py

```diff
@@ -37,7 +37,7 @@ def _get_model_selection(self):
     delay_from = fields.Selection([
         ('current_date', 'after completion date'),
         ('previous_activity', 'after previous activity deadline')], string="Delay Type", help="Type of delay", required=True, default='previous_activity')
-    icon = fields.Char('Icon', help="Font awesome icon e.g. fa-tasks")
+    icon = fields.Char('Icon', help="Font awesome icon e.g. fa-tasks", default="fa-tasks")
```

Comment:
You probably need to make migration scripts, so that old icons without a value now use `fa-tasks`.

Reply:
Ok. I'll do the same.